src/dsp_tools/commands/xmlupload/models/ontology_lookup_models.py

```diff
@@ -75,3 +77,78 @@ def _get_all_properties_from_json(onto_json: list[dict[str, Any]]) -> list[str]:
 
 def _remove_prefixes(ontology_elements: list[str]) -> list[str]:
     return [x.split(":")[1] for x in ontology_elements]
+
+
+AllowedEncodings = Literal["utf8", "xml"]
+
+
+@dataclass
+class TextValueData:
+    resource_id: str
+    property_name: str
+    encoding: AllowedEncodings
+
+
+@dataclass
+class PropertyTextValueEncodingTypes:
+    """
+    This class contains the information
+    which properties have which type of encoding for a TextValue property in the ontology.
+    """
+
+    formatted_text: set[str] = field(default_factory=set)
+    unformatted_text: set[str] = field(default_factory=set)
```

Comment:
To make it more clear:

```suggestion
    formatted_text_props: set[str] = field(default_factory=set)
    unformatted_text_props: set[str] = field(default_factory=set)
```